internal/pkg/otel/samples/darwin/logs_metrics_traces.yml

```diff
@@ -0,0 +1,63 @@
+receivers:
```

Comment:
This file has everything that the above `logs_host_metrics.yml` has, and then some more. Is it valuable to have both?

Reply:
adding traces requires adding a listening port so i think it makes sense for it to be optional